pages/windows/subst.md

```diff
@@ -0,0 +1,15 @@
+# subst
+
+> Associates a path with a drive letter.
```

Comment:
Maybe better to say "Associates a path with a virtual drive letter.". To better signify that these are virtual drive mappings and not actual physical drives.